project1/index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Cute+Font|Lato:300,300i,400,400i,700,700i,900,900i&display=swap"
```

Comment:
Two things here:
1. You should only have one Google Fonts stylesheet `link` -- even if you're using multiple fonts, its possible and recommended to reference them all in a single stylesheet. This helps with your page performance (how fast it loads).
2. You should only reference the fonts you actually use in your final project. So, if you load a bunch of fonts to try them out (totally acceptable, I do this, too), before you turn in the project, you should review your CSS and remove any fonts and font weights you are not currently using.

This means you can replace lines 10-15 with this single element:
```
 <link href="https://fonts.googleapis.com/css?family=Prata|Roboto&display=swap" rel="stylesheet"> 
```
Since the only fonts you're using are Prata and Roboto and you don't use any of the Roboto font weights or styles.